plugins/inputs/traefik/traefik.go

```diff
@@ -0,0 +1,167 @@
+package traefik
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Traefik struct {
+	Address                      string
+	ResponseTimeout              internal.Duration
+	IncludeStatusCodeMeasurement bool
+	lastRequestTiming            float64
+	tls.ClientConfig
+	client *http.Client
+}
+
+type HealthCheck struct {
+	UptimeSec              float64   `json:"uptime_sec"`
+	Unixtime               int64     `json:"unixtime"`
+	TotalStatusCodeCount   HttpCodes `json:"total_status_code_count"`
+	TotalCount             int       `json:"total_count"`
+	TotalResponseTimeSec   float64   `json:"total_response_time_sec"`
+	AverageResponseTimeSec float64   `json:"average_response_time_sec"`
+}
+
+type HttpCodes map[string]int
+
+var sampleConfig = `
+    # Required Traefik server address, host and port (default: "127.0.0.1")
```

Comment:
Should the default be `http://127.0.0.1:8080`? If not, maybe add the default port (`8080` from line 164) to the documentation here and in the readme.